app/src/components/widgets/singleLevelWidgets/charts/passingRateSummary/passingRateSummary.jsx

```diff
@@ -43,11 +44,25 @@ export class PassingRateSummary extends Component {
   };
 
   onChartClick = (data) => {
-    const { widget, getStatisticsLink, project } = this.props;
+    const {
+      widget,
+      getStatisticsLink,
+      project,
+      widget: {
+        contentParameters: {
+          widgetOptions: { includeSkipped },
+        },
+      },
+    } = this.props;
+
+    const linkCreationParametersForFailed = includeSkipped
+      ? [FAILED, INTERRUPTED, SKIPPED]
+      : [FAILED, INTERRUPTED];
+
     const link = getStatisticsLink({
-      statuses: data.id === STATS_PASSED ? [PASSED] : [FAILED, INTERRUPTED, SKIPPED],
+      statuses: data.id === STATS_PASSED ? [PASSED] : linkCreationParametersForFailed,
       launchesLimit: widget.contentParameters.itemsCount,
-      types: null,
+      types: TEST_ITEM_PAGE,
```

Comment:
What is the reason of providing `TEST_ITEM_PAGE` here?

Reply:
As I understand correctly, we need to display only method type 'Test'.
`NOTE: Only method type 'Test' should be displayed.` It is from acceptance criteria.

Reply:
So you can just remove `types` property at all to see the correct result.

Reply:
Function will use `'filter.in.type': LEVEL_STEP,` parameter

Reply:
Ah, yes I see.
It is good remark!